README.md

```diff
@@ -1 +1,167 @@
-# witnet-block-relay
\ No newline at end of file
+# witnet-ethereum-block-relay
+
+`witnet-ethereum-block-relay` is an open source implementation of the Block Relay in Witnet.
+
+ DISCLAIMER: this is a work in progress, meaning the contract could be voulnerable to attacks.
+
+## About WBI and the Block Relay
+
+The Bridge nodes interact through the Witnet Bridge Interface, a smart contract in charge of facilitating the communication between Witnet and samrt contract platmforms, such as Ethereum (click [here][wbi] for more information).
+The block headers in Witnet need to be available to the WBI, meaning the WBI should get those headers that should be stored in some contract, the Block Relay, in Witnet. In order to assure the validity of the block headers, the Block Relay needs to guarantee a consensus protocol and a Finality Gadget. The documentation about the stages of the Block Relay can be found [here][block-relay].
```

Comment:
I'm confused by this sentence, but I think you mean:

The block headers in Witnet must be available to the WBI, so the WBI should retrieve headers which are then stored in a contract (the Block Relay) in Witnet. In order to ensure the validity of the block headers, the Block Relay must have a Consensus Protocol and a Finality Gadget. The documentation regarding the stages of the Block Relay can be found [here][block-relay].